src/CardTheme.res

```diff
@@ -66,6 +66,7 @@ type recoilConfig = {
   config: configClass,
   themeObj: themeClass,
   localeString: LocaleStringTypes.localeStrings,
+  stringConfig: LocaleStringTypes.strings,
```

Comment:
Could you please give a different naming. The meaning is not intuitive by what you mean by stringConfig

Reply:
suggest something pls

Reply:
Renamed it to `strings`